bindata/scripts/kargs.sh

```diff
@@ -5,54 +5,83 @@ command=$1
 shift
 declare -a kargs=( "$@" )
 ret=0
-args=$(chroot /host/ cat /proc/cmdline)
 
-IS_OS_UBUNTU=true; [[ "$(chroot /host/ grep -i ubuntu /etc/os-release -c)" == "0" ]] && IS_OS_UBUNTU=false
+if [[ -d "/bindata/scripts" ]];then
```

Comment:
nit :add comment, its non obvious that in this case we run in from config daemon and the other by system service.

Reply:
done

Reply:
can you check the comment again.  it might have been dropped by a git rebase

Reply:
I don't see the comment also